src/test/regress/expected/upgrade_distributed_table_after.out

```diff
@@ -15,5 +15,125 @@ SELECT * FROM t WHERE a = 1;
  1
 (1 row)
 
+INSERT INTO t SELECT * FROM generate_series(10, 15);
+SELECT * FROM t WHERE a = 10;
+ a  
+----
+ 10
+(1 row)
+
+SELECT * FROM t WHERE a = 11;
+ a  
+----
+ 11
+(1 row)
+
+-- test task-tracker executor
+SET citus.task_executor_type TO 'task-tracker';
+SELECT * FROM t WHERE a = 10;
+ a  
+----
+ 10
+(1 row)
+
+SELECT * FROM t WHERE a = 11;
+ a  
+----
+ 11
+(1 row)
+
+SELECT * FROM t WHERE a < 3 ORDER BY a;
+ a 
+---
+ 1
+ 2
+(2 rows)
+
+RESET citus.task_executor_type;
+-- test adaptive executor
+SET citus.task_executor_type to "adaptive";
+SELECT * FROM t WHERE a = 10;
+ a  
+----
+ 10
+(1 row)
+
+SELECT * FROM t WHERE a = 11;
+ a  
+----
+ 11
+(1 row)
+
+SELECT * FROM t WHERE a < 3 ORDER BY a;
+ a 
+---
+ 1
+ 2
+(2 rows)
+
+RESET citus.task_executor_type;
+-- test real-time executor
+SET citus.task_executor_type to "real-time";
+SELECT * FROM t WHERE a = 10;
```

Comment:
I think testing the different executors after upgrade is probably not very interesting. We're mainly interested in what happens to metadata/catalogs during the upgrade.

Reply:
@marcocitus I thought that we could use this test both in postgres and citus upgrades. But if it doesnt make sense to include this for postgres upgrades I can put this to only citus upgrade tests.